x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -752,6 +756,24 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
                           )}
                         />
                       </EuiFlexItem>
+                      {isAdHocRuleExecutionEnabled && (
```

Comment:
when I tried to run rule, got this error

<img width="2396" alt="Screenshot 2023-06-09 at 10 21 35" src="https://github.com/elastic/kibana/assets/92328789/19784c5b-0012-4e2d-bceb-297f243ca4e3">